From 61191591433ae414845a3356225431618803c5bc Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty
 <45927311+Etaash-mathamsetty@users.noreply.github.com>
Date: Wed, 5 Aug 2026 18:17:14 -0400
Subject: [PATCH 266/268] winewayland: Don't activate when restoring window.

setting the window to foreground already does that.
---
 dlls/winewayland.drv/window.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/winewayland.drv/window.c b/dlls/winewayland.drv/window.c
index 96a0c9a8b4c..887ee49fd14 100644
--- a/dlls/winewayland.drv/window.c
+++ b/dlls/winewayland.drv/window.c
@@ -954,7 +954,7 @@ BOOL WAYLAND_GetWindowStateUpdates(HWND hwnd, UINT *state_cmd, UINT *swp_flags,
     /* we can't track if the host window is minimized or unminimized, but
      * if we have keyboard focus on this window we can treat it as restored. */
     if ((style & WS_MINIMIZE) && focused_hwnd == hwnd && old_foreground != hwnd)
-        *state_cmd = MAKELONG(SC_RESTORE, 1);
+        *state_cmd = MAKELONG(SC_RESTORE, 0);
 
     TRACE("hwnd=%p foreground=%p state=%#x\n", hwnd, *foreground, *state_cmd);
 
-- 
2.55.0

